test/invariant/fuzz/handlers/HandlerAllo.t.sol

```diff
@@ -224,7 +230,7 @@ contract HandlerAllo is Setup {
     function _pickPoolId(uint256 _idSeed) internal view returns (uint256) {
         if (ghost_poolIds.length == 0) return 0;
 
-        return ghost_poolIds[_idSeed % ghost_poolIds.length];
+        return ghost_poolIds[_idSeed % ghost_poolIds.length - 1];
```

Comment:
the `-1` seems incorrect to me. `idSeed` starts at 0, meaning seed % length starts at 0 too (and goes up to `length - 1` when `seed = length-1` (when seed = length, then length%length is back to the indice 0 of the array)

Reply:
it's not reverting during the test as it's outside of an assert, but leave the last element of the array untouched

Reply:
good catch, will fix